app/routes/index.js

```diff
@@ -2,6 +2,6 @@ import Route from '@ember/routing/route';
 
 export default Route.extend({
     beforeModel() {
-        this.replaceWith('users');
+        this.replaceWith('users/new');
```

Comment:
Why do we need this ?

Reply:
Its not needed. Updated with previous one.